app/employee-app/backend/modules/authorization/authorization.bal

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Old Licenser header year and WSO2 LLC has to be updated

Reply:
Done and resolved